rllib/evaluation/tests/test_trajectory_view_api.py

```diff
@@ -717,6 +711,78 @@ def analyze_rnn_batch(batch, max_seq_len, view_requirements):
         cursor += max_seq_len
 
 
+def analyze_rnn_batch_rlm(batch, max_seq_len, view_requirements):
+    # Note that this method assumes a batch with a time dimension
+    count = batch.count
```

Comment:
This is the same as the other analyze-method above, only that some of the assumptions are removed about how neighboring items relate to the others. These would have to sometimes cross sequence-boundaries.